user_guide_src/source/incoming/routing/071.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Config\Services;
+
+// Get the router instance
+$router = Services::router();
```

Comment:
`service()` is recommended. See #8928

```suggestion
$router = service('router');
```